### PR TITLE
Case 21160: Tablet does not properly resize when scaling avatar. 

### PR DESCRIPTION
**Pull request by danteruiz**
Ticket - https://highfidelity.manuscript.com/f/cases/21160/Tablet-does-not-properly-resize-when-scaling-avatar

### CODE REVIEW

File: scripts/system/libraries/utils.js [hunk 1]
```diff
@@ -407,7 +407,7 @@ resizeTablet = function (width, newParentJointIndex, sensorToWorldScaleOverride)
     var landscape = Tablet.getTablet("com.highfidelity.interface.tablet.system").landscape;
     Overlays.editOverlay(HMD.tabletScreenID, {
         localPosition: { x: 0, y: WEB_ENTITY_Y_OFFSET, z: -WEB_ENTITY_Z_OFFSET},
-        dimensions: {x: landscape ? screenHeight : screenWidth, y: landscape ? screenWidth : screenHeight, z: 0.1},
```
Comment (SamGondelman) [inline scripts/system/libraries/utils.js]: were these necessary?

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17611/HighFidelity-Beta-PR15024-b6a45a2.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17611/HighFidelity-Quest-Beta-PR15024-b6a45a2.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17611/HighFidelity-Beta-PR15024-b6a45a2.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17611/HighFidelity-Beta-Interface-PR15024-b6a45a2.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17611/HighFidelity-Beta-PR15024-b6a45a2.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17611/HighFidelity-Beta-Interface-PR15024-b6a45a2.exe

**Comment by hifi-gustavo** [comment]
Can someone from High Fidelity approve this PR?

File: scripts/system/libraries/utils.js (hunk 1 above)
Comment (SamGondelman) [reply scripts/system/libraries/utils.js]: talked with dante, will address in master

**Pull request closed without merging**